template/src/ontology/Makefile.jinja2

```diff
@@ -426,14 +426,12 @@ prepare_release: all_odk
 	rm -f $(CLEANFILES) &&\
 	echo "Release files are now in $(RELEASEDIR) - now you should commit, push and make a release \
         on your git hosting site such as GitHub or GitLab"
+{%- if project.commit_release_artefacts %}
 
 .PHONY: prepare_initial_release
-prepare_initial_release: all_assets
-	rsync -R $(RELEASE_ASSETS) $(RELEASEDIR) &&\{% if project.sssom_mappingset_group is defined %}{% if project.sssom_mappingset_group.released_products is defined %}
-	mkdir -p $(RELEASEDIR)/mappings && cp -rf $(RELEASED_MAPPING_FILES) $(RELEASEDIR)/mappings &&\{% endif %}{% endif %}{% if project.use_dosdps %}
-	mkdir -p $(RELEASEDIR)/patterns && cp -rf $(PATTERN_RELEASE_FILES) $(RELEASEDIR)/patterns &&\{% endif %}
-	rm -f $(patsubst %, ./%, $(CLEANFILES)) &&\
-	cd $(RELEASEDIR) && git add $(RELEASE_ASSETS)
+prepare_initial_release: prepare_release
+	git add $(foreach asset,$(RELEASE_ASSETS),$(RELEASEDIR)/$(asset))
```

Comment:
Second thought, why not drop the `prepare_initial_release` entirely in favour of a line the odk.py? This is also a noisy goal, once used, never used again, so belongs conceptually in odk.py?

Reply:
My concern with that is that it requires duplicating a logic that currently only exists in the Makefile, but maybe it's not so much of a big deal, I'll have another look.